.travis.yml

```diff
@@ -6,7 +6,6 @@ branches:
 python:
  - "3.6"
  - "3.5"
- - "3.4"
 env:
  - DJANGO_VERSION=1.8.17
  - DJANGO_VERSION=1.10.5
```

Comment:
Do these need to be bump to account for recent security releases too (e.g., 1.10 is up to 1.10.7 and 1.11 just came out to)?